### PR TITLE
webgl_mirror_nodes update

### PR DESCRIPTION
**Pull request by sunag**
https://github.com/mrdoob/three.js/commit/f3454f5809b8f658cdc79ec955ce739f815684fc#commitcomment-21056997

continuing...

5. `setDrawRange` was one of the solutions I found. I believe that one `prevent method` be the best option but the current (4.) is not working.

### CODE REVIEW

File: examples/webgl_mirror_nodes.html [hunk 1]
```diff
@@ -153,7 +152,7 @@
 				var planeGeo = new THREE.PlaneBufferGeometry( 100.1, 100.1 );
 
 				// MIRROR planes
-				groundMirror = new THREE.Mirror( renderer, camera, { clipBias: 0.003, textureWidth: WIDTH, textureHeight: HEIGHT, color: 0x777777 } );
+				var groundMirror = new THREE.Mirror( 100, 100, { clipBias: 0.003, textureWidth: WIDTH, textureHeight: HEIGHT, render: false } );
```
Comment (mrdoob) [inline examples/webgl_mirror_nodes.html]: How about just adding this line here?

```js
groundMirror.geometry.setDrawRange( 0, 0 ); // avoid rendering geometry
```

Comment (sunag) [inline examples/webgl_mirror_nodes.html]: Ok! The try is to make it a little more elegant for others devs if need it.

Comment (mrdoob) [inline examples/webgl_mirror_nodes.html]: Yep yep. I understand... Lets expose the hack outside for now 😅

Comment (sunag) [inline examples/webgl_mirror_nodes.html]: Updated! :+1:

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
Hmm... Almost there!

screen shot 2017-03-06 at 10 25 22

**Comment by sunag** [comment]
My first try to skip the `groundMirrorMaterial` render. 
(Warns fixed but it not "understanding" the scene lights)

```javascript
groundMirror._onBeforeRender = groundMirror.onBeforeRender; // create a super function
groundMirror.onBeforeRender = function( renderer, scene, camera ) {

	// avoid rendering mirrorMaterial
	// true = work done but with warns
	// false = not recognizes the scene lights
	groundMirrorMaterial.visible = false; 

	this._onBeforeRender( renderer, scene, camera );

	groundMirrorMaterial.visible = true;

};
```

I try using `THREE.Layers` but the result was the same.

light-not-rendering

**Pull request merged**